src/main/java/com/xiaomi/infra/pegasus/rpc/ThriftHeader.java

```diff
@@ -6,32 +6,18 @@
 import java.nio.ByteBuffer;
 
 public class ThriftHeader {
-  public static final int HEADER_LENGTH = 48;
+  public static final int HEADER_LENGTH = 16;
   static final byte[] HEADER_TYPE = {'T', 'H', 'F', 'T'};
-  public int hdr_version = 0;
```

Comment:
Is it safety to remove varieties such as `hdr_version`, `header_crc32`,  `body_crc32 `, `thread_hash` directly?

Reply:
same as above